manifests/mount.pp

```diff
@@ -31,6 +31,7 @@
       exec {"create ${mountpoint} and parents":
         command => "mkdir -p ${mountpoint}",
         unless  => "test -d ${mountpoint}",
+        path    => ['/usr/bin','/bin'],
```

Comment:
This should be in a different patch an you should maybe use `path => $::path` instead